src/Alert.js

```diff
@@ -0,0 +1,69 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+
+const FirstChild = ({ children }) => (
+  React.Children.toArray(children)[0] || null
+);
+
+const propTypes = {
+  className: PropTypes.any,
+  color: PropTypes.string,
+  isOpen: PropTypes.bool,
+  toggle: PropTypes.func,
+  tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string])
+};
+
+const defaultProps = {
+  color: 'success',
+  isOpen: true,
+  tag: 'div'
+};
+
+const Alert = (props) => {
+  const {
+    className,
+    tag: Tag,
+    color,
+    isOpen,
+    toggle,
+    children,
+    ...attributes
+  } = props;
+
+  const classes = classNames(
+    className,
+    'alert',
+    `alert-${color}`,
+    { 'alert-dismissible': toggle }
+  );
+
+  const alert = (
+    <Tag {...attributes} className={classes} role="alert">
+      { toggle ?
+        <button type="button" className="close" aria-label="Close" onClick={toggle}>
+          <span aria-hidden="true">&times;</span>
+        </button>
+        : null }
+      { children }
+    </Tag>
+  );
+
+  return (
+    <ReactCSSTransitionGroup
+      component={FirstChild}
+      transitionName={{
```

Comment:
experimented with this, the following example seems to apply the right classes and works with the default opacity animation of `.15s`. 

```jsx
<ReactCSSTransitionGroup
  component={FirstChild}
  transitionName={{
    enter: 'fade',
    enterActive: 'in',
    leave: 'fade',
    leaveActive: 'out',
    appear: 'fade',
    appearActive: 'in'
  }}
  transitionEnterTimeout={150}
  transitionLeaveTimeout={150}
>
  {isOpen ? alert : null}
</ReactCSSTransitionGroup>
```